app/uk/gov/hmrc/merchandiseinbaggage/connectors/MibConnector.scala

```diff
@@ -51,4 +52,7 @@ class MibConnector @Inject()(httpClient: HttpClient, @Named("mibBackendBaseUrl")
     httpClient
       .POST[String, HttpResponse](s"$base$sendEmailsUrl/${declarationId.value}", "")
       .map(_ => ())
+
+  def checkEoriNumber(eori: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[CheckResponse] =
+    httpClient.GET[CheckResponse](s"$base$checkEoriUrl$eori")
```

Comment:
do we need `base` here ?

Reply:
not sure - i think Guice is using it to ovveride it in fakeApp - I am proudly NOT a Guice expert